myhpi/static/css/myHPI.css

```diff
@@ -203,4 +203,39 @@ nav .nav-link[aria-expanded=true] {
 h1.toc-title::after {
     margin: 0;
     height: 0px;
-}
\ No newline at end of file
+}
+
+.print-shown {
+    display: none;
+}
+
+@media print {
+    .print-none {
+        display: none !important;
+    }
+
+    .print-shown {
+        display: block !important;
+    }
+
+    nav, .toc, .toc-container {
+        display: none;
+    }
+
+    .minutes-container {
+        display: flex;
+        flex-direction: column;
+    }

```

Comment:
_Could_ be done with the bootstrap utilities as well, but giving the container some semantic is fine to me, too